docs/sphinx/install.rst

```diff
@@ -3,16 +3,38 @@
 Installation
 ============
 
-The histogram package can be installed by using pip install::
+The histogram package can be installed with 2 ways:
 
- Download the source code from https://github.com/neutrons/histogram-dev.
+* from the source code:
 
- To install:
+    1. Install `Pixi <https://pixi.sh/>`_ for managing environments, dependencies, packaging, and task execution. (if it is not already installed).
+
+        Pixi installation e.g. for Linux:
+
+        .. code:: shell
+
+            $ curl -fsSL https://pixi.sh/install.sh | sh
+
+
+        Also install git, if it is missing.
```

Comment:
Also install git, if it is missing, on Linux Ubuntu:
..code:: shell

$ sudo apt update
$ sudo apt install git.